bin/generate_rest.sh

```diff
@@ -50,5 +50,12 @@ echo "...generating python sdk client"
 swagger-codegen generate -i specs/zrok.yml -o sdk/python/sdk/zrok -c $pythonConfig -l python
 
 git checkout rest_server_zrok/configure_zrok.go
-rm sdk/nodejs/sdk/src/zrok/api/git_push.sh
-rm sdk/python/sdk/zrok/git_push.sh
+
+for OLD in sdk/python/sdk/zrok/git_push.sh sdk/nodejs/sdk/src/zrok/api/git_push.sh
+do
+  if [ -e "$OLD" ]
+  then
+    rm "$OLD"
+  fi
+done
+
```

Comment:
why not just `rm -f` instead of `rm`?

Reply:
I need to understand their function to handle them correctly. 

Is it an error if these `git_push.sh` do not exist? Where are they from? What do they do?

Reply:
I added this conditional delete because one, but not both, existed.

Reply:
They're generated artifacts that should not be checked in.